charts/osm/values.schema.json

```diff
@@ -320,6 +320,15 @@
                         false
                     ]
                 },
+                "enablePrometheusScraping": {
+                    "$id": "#/properties/OpenServiceMesh/properties/enablePrometheusScraping",
+                    "type": "boolean",
+                    "title": "The enablePrometheusScraping schema",
```

Comment:
Should I add "Deprecated" in the title and/or the description?

Reply:
Yes, please.